static/configuration.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+mkdir -p "$SCRIPTS"
+print_text_in_color "$ICyan" "Running the nextcloud configuration script..."
+
+if network_ok
+then
+    # Delete, download, run
+    run_static_script nextcloud_configuration
```

Comment:
`run_static_script` first deletes all copies of the script, then it downloads it, then it runs it, and then deletes it again. 

The whole point here is that everything should be in the setup script already. Maybe I was too quick merging your other PR, not when I think of it, it actually made things ore unstable.

Reply:
yes, I think that the configuration script isn't that important. Since you get additional options, that aren't really neccessary to run nextcloud but that are welcome additions.